02-getting_started.Rmd

```diff
@@ -27,190 +27,315 @@ The `r-devel` at [R/trunk](https://svn.r-project.org/R/trunk "svn devel source c
-          bash-completion \
-          bison \
-          debhelper-compat (= 11) \
-          default-jdk \
-          g++ (>= 4:4.9.2-2) \
-          gcc (>= 4:4.9.2-2) \
-          gfortran (>= 4:4.9.2-2) \
-          groff-base \
-          libblas-dev \
-          libbz2-dev \
-          libcairo2-dev \
-          libcurl4-dev \
-          libcurl4-openssl-dev \
-          libjpeg-dev \
-          liblapack-dev \
-          liblzma-dev \
-          libncurses5-dev \
-          libpango1.0-dev \
-          libpcre2-dev \
-          libpcre3-dev \
-          libpng-dev \
-          libreadline-dev \
-          libtiff5-dev \
-          libx11-dev \
-          libxt-dev \
-          mpack \
-          tcl8.6-dev \
-          texinfo (>= 4.1-2) \
-          texlive-base \
-          texlive-extra-utils \
-          texlive-fonts-extra \
-          texlive-fonts-recommended \
-          texlive-latex-base \
-          texlive-latex-extra \
-          texlive-latex-recommended \
-          texlive-plain-generic \
-          tk8.6-dev \
-          x11proto-core-dev \
-          xauth \
-          xdg-utils \
-          xfonts-base \
-          xvfb \
-          zlib1g-dev
-
-
-### Windows
-
-Install MSYS and rtools42
-
-### macOS
 
-
-## Building R in Linux
+```sh
+sudo apt-get install -y \
+      bash-completion \
+      bison \
+      debhelper-compat \
+      default-jdk \
+      g++ \
+      gcc \
+      gfortran \
+      groff-base \
+      libblas-dev \
+      libbz2-dev \
+      libcairo2-dev \
+      libcurl4-dev \
+      libcurl4-openssl-dev \
+      libjpeg-dev \
+      liblapack-dev \
+      liblzma-dev \
+      libncurses5-dev \
+      libpango1.0-dev \
+      libpcre2-dev \
+      libpcre3-dev \
+      libpng-dev \
+      libreadline-dev \
+      libtiff5-dev \
+      libx11-dev \
+      libxt-dev \
+      mpack \
+      tcl8.6-dev \
+      texinfo \
+      texlive-base \
+      texlive-extra-utils \
+      texlive-fonts-extra \
+      texlive-fonts-recommended \
+      texlive-latex-base \
+      texlive-latex-extra \
+      texlive-latex-recommended \
+      texlive-plain-generic \
+      tk8.6-dev \
+      x11proto-core-dev \
+      xauth \
+      xdg-utils \
+      xfonts-base \
+      xvfb \
+      zlib1g-dev 
+```
+
+### Fedora
+
+<!-- # Command to obtain it: dnf rq -q --repo=fedora-source --requires R -->
+In Fedora you can use:
+
+```sh
+sudo dnf install -y \
+    autoconf \
+    automake \
+    bzip2-devel \
+    cairo-devel \
+    flexiblas-devel \
+    gcc-c++ \
+    gcc-gfortran \
+    java-devel \
+    less \
+    libICE-devel \
+    libSM-devel \
+    libX11-devel \
+    libXmu-devel \
+    libXt-devel \
+    libcurl-devel \
+    libicu-devel \
+    libjpeg-devel \
+    libpng-devel \
+    libtiff-devel \
+    libtool \
+    ncurses-devel \
+    pango-devel \
+    pcre2-devel \
+    readline-devel \
+    tcl-devel \
+    'tex(inconsolata.sty)' \
+    'tex(latex)' \
+    'tex(upquote.sty)' \
+    'texinfo-tex' \
```

Comment:
Not this one. Sorry, I wasn't clear enough: only the ones that have parentheses require quoting.